taurus/ingest/material_run_example.py

```diff
@@ -20,7 +20,7 @@
     ),
     "kinematic viscosity": PropertyTemplate(
         name="kinematic viscosity",
-        bounds=RealBounds(lower_bound=0.0, upper_bound=10.0**40, default_units="m^2/s")
+        bounds=RealBounds(lower_bound=0.0, upper_bound=10.0**40, default_units="")
```

Comment:
Why is this non-dimensional?  Meter and second are supported.

Reply:
Because the parsing logic that I'm using in this example is terrible and gets stuck on spaces.  I can just cram the units together though.  Update coming.

Reply:
Do we need the PR in before the update is ready?

Reply:
No; I appended to this PR.